apps/playground-api/src/modules/PlaygroundModule.ts

```diff
@@ -66,8 +65,7 @@ export class PlaygroundModule implements OnApplicationBootstrap {
       loanScheme,
       loanToken,
       loanCollateral,
-      dex,
-      gov
+      dex
```

Comment:
Actually, why do we need it to run after `SetupLoanToken`, rather we can have this run as its' own "Setup" since the setup service run sequentially?

Reply:
yes that's few approaches.. early was thinking to take tokenId from memory instead of rpc.. but find out its not possible..
yea will move back to `setup` flow